topologies/proto/binding.proto

```diff
@@ -27,6 +27,18 @@ message Binding {
   Options options = 3;
 }
 
+// Device Config to be passed during Device binding
+message Configs {
+  // Raw device config
+  repeated bytes cli_config_data = 1;
```

Comment:
In retrospect, let's call this `cli` and `cli_file` to avoid repeating `config` too many times, and `data` is kind of meaningless. Currently:

```
duts {
  config {
    cli_config_data: "..."
    cli_config_data: "..."
    cli_config_file: "..."
  }
}
```

After:

```
duts {
  config {
    cli: "..."
    cli: "..."
    cli_file: "..."
  }
}
```

Looks better after. What do you think?

Reply:
Done!